KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/TemplateControlModule.java

```diff
@@ -136,10 +136,17 @@ public void onInitAfterLoadTemplate(LiveStreamTranscoder liveStreamTranscoder) {
             TranscoderStream transcoderStream =  liveStreamTranscoder.getTranscodingStream();
             java.util.List<com.wowza.wms.transcoder.model.TranscoderStreamDestination> transcoderStreamsMap =  transcoderStream.getDestinations();
             IMediaStream stream = liveStreamTranscoder.getStream();
+
             if (transcoderStreamsMap.size() == 0){
                 String msg = "Stream " + stream.getName() + " has no ingest in conversion profile";
                 logger.error(msg);
-                DiagnosticsProvider.addRejectedStream(msg, stream.getClient());
+                if (stream.getClient() != null){
```

Comment:
i think it's better to move this logic inside the DiagnosticsProvider (DiagnosticsProvider.addRejectedStream(ImediaStream, msg)